kafka/integration_test.go

```diff
@@ -552,6 +622,18 @@ func TestProducerFuncDR(t *testing.T) {
 		})
 }
 
+// test producer function-based API with stats and delivery report
```

Comment:
As with librdkafka I'd like to see a unittest for the stats that doesnt rely on broker connectivity.